db/migrate/20201210130513_add_disability_information_to_trainee_disabilities.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDisabilityInformationToTraineeDisabilities < ActiveRecord::Migration[6.0]
+  def change
+    add_column :trainee_disabilities, :additional_disability, :text, null: true
```

Comment:
Does this `null: true` do anything for columns that aren't foreign keys?

Reply:
So this field is a bit weird but we decided to keep it on the join table. This table can hold extra information about the relationship between a trainee and a disability. It doesn't make sense to keep it on the `Disabilities` table.